scripts/protocgen.sh

```diff
@@ -8,7 +8,7 @@ protoc_gen_gocosmos() {
     return 1
   fi
 
-  go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest 2>/dev/null
+  go install github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@latest 2>/dev/null
```

Comment:
This is not working with the docker image (tendermintdev/sdk-proto-gen) that we use. Can you revert this change/ test with `make proto-gen` ?